src/app/showcase/tabs-showcase/tabs-showcase.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+
+import { ShowcaseProperty } from '../../shared/showcase-properties/showcase-property';
+
+@Component({
+  selector: 'kirby-tabs-showcase',
+  templateUrl: './tabs-showcase.component.html',
+  styleUrls: ['./tabs-showcase.component.scss'],
+})
+export class TabsShowcaseComponent implements OnInit {
+  exampleHtml: string = `<kirby-tab-bar>
```

Comment:
Consider grabbing the html from the actual example to make sure they are always in sync. See PageShowcaseComponent/PageSimpleExampleComponent.template for an example